app/views/FileTooLargeView.scala.html

```diff
@@ -28,7 +28,12 @@
 
     <h1 class="govuk-heading-l">@messages("fileTooLarge.heading")</h1>
 
-    <p class="govuk-body">@messages("fileTooLarge.p1")</p>
+    <p class="govuk-body">@Html(messages("fileTooLarge.p1", link("mailto:aeoi.enquiries@hmrc.gov.uk", "fileTooLarge.link")))</p>
```

Comment:
can you make the email address a config value in application.conf and use the same value for the link text rather than a message